storm/clean_externals.py

```diff
@@ -34,17 +35,18 @@ def main():
 
     invalid = keep.difference(externals)
     if not keep.issubset(externals):
-        print('Invalid values for KEEP_EXTERNALS: %r' % invalid,
-              file=sys.stderr)
+        print('Invalid values for KEEP_EXTERNALS: {!r}'
+              .format(invalid, file=sys.stderr))
```

Comment:
`file=` should be outside the inner parens ... also not totally sure why '%' is preferred?